packages/presentation/presentation_publishing/publishing_subject/publishing_batcher.ts

```diff
@@ -27,10 +27,15 @@ const hasSubjectsArrayChanged = (
 /**
  * Batches the latest values of multiple publishing subjects into a single object. Use this to avoid unnecessary re-renders.
  * You should avoid using this hook with subjects that your component pushes values to on user interaction, as it can cause a slight delay.
+ * @param subjects Publishing subjects array.
+ *   When 'subjects' is expected to change, 'subjects' must be part of component react state.
+ *   State change required for component to re-render.
```

Comment:
nit: I'm not sure these two lines are necessary. IMO you've summed it up well enough with `When 'subjects' array is expected to change, 'subjects' must be part of component react state`

Reply:
right. The more I think about it, this is standard react fare, so no need to make a big deal about this requirement. I will remove the last 2 lines of the comment.